src/session/model/SessionInterfaces.ts

```diff
@@ -21,6 +21,7 @@ export type ISession = {
 export type ISignInInfo = {
     [SignInInfoKeys.AccessToken]?: IAccessToken,
     [SignInInfoKeys.AdminPermissions]?: string,
+    [SignInInfoKeys.CompanyNumber]?: string,
```

Comment:
I have never seen company number in sign in info? How is that populated?

Reply:
after hitting ACCOUNT_SERVICE/oauth2/company/signin? endpoint and providing the company auth code, the companyNumber is set in the session to confirm that the user is authorized to act on behalf of the company.